sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Tests/LiveMetrics/DocumentTests/HttpClientDependecyTests.cs

```diff
@@ -69,8 +69,7 @@ public void VerifyHttpClientAttributes()
             Assert.Equal(DocumentType.RemoteDependency, dependencyDocument.DocumentType);
             Assert.Equal("HelloWorld", dependencyDocument.Name);
             Assert.Equal("200", dependencyDocument.ResultCode);
-
-            VerifyCustomProperties(dependencyDocument);
```

Comment:
I don't agree with this change. This method verifies lines 52 - 62.

Reply:
The idea is that we should not control what gets added to the properties in live metrics. Any check on it is a waste of CPU resources.

Reply:
We should discuss in standup.

I checked out this PR and the behavior is unchanged (ie: this Verify method would pass).

I agree in theory that we shouldn't waste CPU resources.
But a change in behavior here would only affect Dependencies. All other telemetry types are selectively choosing properties. My understanding is that these documents should be representative of the actual telemetry being ingested. We can discuss with service team if we need to clarify the requirements.

Reply:
Maybe I'm misunderstanding your comment.

This if/else appears to be only adding properties to the document that aren't in the semanticsSet.
https://github.com/Azure/azure-sdk-for-net/blob/e1b3388976dc10ac69718cf5e14b40d06c08425d/sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Internals/LiveMetricsTagsProcessor.cs#L48-L67

Reply:
Reverted the test change.